R/Renderer.R

```diff
@@ -57,16 +60,24 @@ Renderer <- R6::R6Class( # nolint: object_name_linter.
     #'
     #' yaml_header <- teal.reporter:::md_header(yaml::as.yaml(yaml_l))
     #' result_path <- teal.reporter:::Renderer$new()$renderRmd(reporter$get_blocks(), yaml_header)
-    renderRmd = function(blocks, yaml_header) {
+    renderRmd = function(blocks, yaml_header, global_knitr = list()) {
       checkmate::assert_list(blocks, c("TextBlock", "PictureBlock", "NewpageBlock", "TableBlock", "RcodeBlock"))
       if (missing(yaml_header)) {
         yaml_header <- md_header(yaml::as.yaml(list(title = "Report")))
       }
-
       parsed_yaml <- yaml_header
-      parsed_blocks <- paste(unlist(lapply(blocks, function(b) private$block2md(b))), collapse = "\n\n")
+      parsed_global_knitr <- sprintf(
+        "\n```{r setup, include=FALSE}\noptions(width = 80)\nknitr::opts_chunk$set(%s)\n```\n",
```

Comment:
```suggestion
        "\n```{r setup, include=FALSE}\nknitr::opts_chunk$set(%s)\n```\n",
```